### PR TITLE
add invoker resources and add couchdb pvc

### PR DESCRIPTION
**Pull request by daisy-ycguo**
Hi, Tom

Please review. I made following changes:
- add resources for invoker
- add pvc for couchdb

Daisy

### CODE REVIEW

File: charts/couchdb/templates/deployment.yaml [hunk 1]
```diff
@@ -32,3 +32,10 @@ spec:
           value: {{ .Values.global.couchdbPassword | default "some_passw0rd" | quote }}
         - name: "DB_PORT"
           value: {{ include "couchdb_port" . | quote }}
+        volumeMounts:
```
Comment (xingzhou) [inline charts/couchdb/templates/deployment.yaml]: It's better to check the `persistence.enabled` option here as well. If not specified, do not need this section

File: charts/couchdb/templates/pv.yaml [hunk 2]
```diff
@@ -0,0 +1,15 @@
+apiVersion: v1
```
Comment (xingzhou) [inline charts/couchdb/templates/pv.yaml:1]: I don't think using PV here is the best way to use a persistent store. As using PV, need to config the volume source/type and detail configs(e.g. NFS path). It's better to use a dynamic provisioner to do so, which means except for a PVC definition, we also need a storage class definition.

**Comment by xingzhou** [comment]
Daisy, some comments for this PV support, FYI

**Comment by daisy-ycguo** [comment]
I addressed your first comments.

**Comment by xingzhou** [comment]
ok, let me resolve the second one

**Pull request merged**